firmware/sys/at_client/doc.txt

```diff
@@ -0,0 +1,36 @@
+/**
+@ingroup     sys
+
```

Comment:
or something like this
```suggestion
@defgroup at_uart_firm
```
*** Be careful to the group defined is not used in wifi-subsys ***